### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by ravindra3003**
Added more info about the verbose option: when to use it and how to use it.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -103,6 +103,22 @@ If you run into issues where running `npx react-native init` hangs in your syste
...
+
+```shell
+npx react-native run-android --verbose
+```
+
+For iOS
+
+```shell
+npx react-native run-ios --verbose
+```
```
Comment (cortinico) [inline docs/troubleshooting.md:120]: Can we merge them into a single snippet?

```
// Android
npx react-native run-android --verbose
// iOS
npx react-native run-ios --verbose
```

Comment (ravindra3003) [reply docs/troubleshooting.md:120]: We can do something like this,

```shell
npx react-native run-android --verbose && npx react-native run-ios --verbose
```

Also, we can add note like this :

If you only need to run for android, remove `&& npx react-native run-ios --verbose` and for iOS, remove `npx react-native run-android --verbose &&`

Comment (cortinico) [reply docs/troubleshooting.md]: > If you only need to run for android, remove `&& npx react-native run-ios --verbose` and for iOS, remove `npx react-native run-android --verbose &&`

It's safe to assume that the user knows that `run-android` should be replaced to `run-ios` if they're running an iOS app so I won't be so detailed

Comment (ravindra3003) [reply docs/troubleshooting.md]: Yes, it's easy to assume,  so we can do only this,

```shell
npx react-native run-android --verbose
```

Comment (cortinico) [reply docs/troubleshooting.md]: Yup that would also work

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -103,6 +103,18 @@ If you run into issues where running `npx react-native init` hangs in your syste
 npx react-native init --verbose
 ```
 
+When you're debugging a process or need to know a little more about the error being thrown, they may want to use the verbose option to dump out more logs and information to nail down their issue.
```
Comment (leotm) [inline docs/troubleshooting.md:106]:
```suggestion
When you're debugging a process or need to know a little more about the error being thrown, you may want to use the verbose option to output more logs and information to nail down your issue.
```

**Review by leotm — COMMENTED** [review]
LGTM after the suggested change (let's keep grammar/tense consistent)

<s>Then either @cortinico's [suggestion](https://github.com/facebook/react-native-website/pull/3174#discussion_r905256627) or [yours](https://github.com/facebook/react-native-website/pull/3174#discussion_r907005898) 👍</s>

**Review by cortinico — APPROVED** [review]

**Pull request merged**